alacritty/src/display/hint.rs

```diff
@@ -223,6 +223,25 @@ impl HintMatch {
     }
 }
 
+/// A hint being highlighted by the mouse or vi cursor.
+#[derive(PartialEq, Eq)]
+pub struct HighlightedHint {
+    pub last_cell_id: u32,
```

Comment:
I might call this `last_end_cell_id` to make it clear where this points and a bit more clear what this does.

Reply:
That's what the `last` is for. It's not indicating that the value is historic. So if anything it should be `end_cell_id`?

Reply:
Yea I think I was sorta misinterpreting it at first. I like calling it `end_cell_id` since it's called `end()` where it's fetched. But I don't care much either way.